rtree/nearest.go

```diff
@@ -6,11 +6,7 @@ import "container/heap"
 // as measured by the Euclidean metric. Note that there may be multiple records
 // that are equidistant from the input box, in which case one is chosen
 // arbitrarily. If the RTree is empty, then false is returned.
-func (t *RTree) Nearest(box Box) (int, bool) {
```

Comment:
not caught by linter but noticed it could be simplified.

Reply:
According to [CodeReviewComments](https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters), they reckon it's better _not_ to use named result parameters in order just to avoid declarations inside the function. I agree with that.

_However_, they also say it's ok to use named result parameters to add clarity to the meaning of the return parameters when it's not clear. It could be argued that it's not clear from the method signature than an `int` is a record ID.

This change is good due to the second reason, but (IMO) not the first reason. I'm happy to keep it in. Thoughts?

Reply:
All fair points, agreed 👍